lib/async_hooks.js

```diff
@@ -42,6 +54,8 @@ const { kInit, kBefore, kAfter, kDestroy, kTotals, kCurrentAsyncId,
         kCurrentTriggerId, kAsyncUidCntr,
         kInitTriggerId } = async_wrap.constants;
 
+// Symbols used to store the respective ids on both AsyncResource instances and
+// internal resources.
```

Comment:
yes, but should also add that they'll be added to _any_ object that enters the internal async path; because users can pass arbitrary objects instead of using internally constructed ones.